### PR TITLE
added description for compilion from source

### LINKED ISSUE #12: add description for compilation from source
The getting started guide is missing the part describing how to compile Mainflux from source

Best Regards,
Jovan

### PR DESCRIPTION
**Pull request by chombium**
this pull request fixes #12

Best regards,
Jovan

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -21,6 +21,22 @@ Then start the Docker composition:
 docker-compose up
 ```
 
+From sources:
+
+To download all the sources, and place them in appropriate locations (i.e. $GOPATH), use the
+[installation script](https://github.com/mainflux/mainflux/blob/master/bin/mainflux-install.sh). Once it completes, the script will provide the
+instructions on how to finish the manual installation (i.e. install the required infrastructure).
+
+Please note that the CoAP server is not installed with the installation script, but the CoAP server
```
Comment (drasko) [inline docs/getting-started.md]: CoAP server should be add to Makefile.

**Comment by chombium** [comment]
removed the remarks about the CoAP server as it was added back to the Makefile and the docker-compose

**Comment by drasko** [comment]
LGTM

**Comment by drasko** [comment]
Merged, thanks!

**Pull request merged**